packages/cf-layout/usage.md

```diff
@@ -432,7 +130,7 @@ When using the modifiers described below to create columns,
 the columns will remain stacked for smaller screens and then convert to to
 columns at `801px`.
 
-`.content_bar` must come after `.content_hero` (if it exists) but before
+`.content_line` must come after `.content_hero` (if it exists) but before
```

Comment:
I think this sentence, and the two `<div class="content_line"></div>` just below, can be deleted.

Reply:
Removed in https://github.com/cfpb/capital-framework/pull/931/commits/ec5377ff8fc3a9b0a0b189e0b2575ad27cb7caba